<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeMakerUIClass</class>
 <widget class="QMainWindow" name="VolumeMakerUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VolumeMakerUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="MpuViewer" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>38</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Volume</string>
     </property>
     <widget class="QSpinBox" name="sbXres">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbYres">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbZres">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QSlider" name="hsXclip_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsYclip_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsZclip_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsXclip_e">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsYclip_e">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsZclip_e">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbVolume2Mesh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume2Mesh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMesh2Volume">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mesh2Volume</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>ViewSamples</string>
     </property>
     <widget class="QSpinBox" name="sbViewId">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>viewId</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbGenViews">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GenViews</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbBatchConfig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>700</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>BatchConfig</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>hdf5</string>
     </property>
     <widget class="QSpinBox" name="sbHdf5Idx">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MpuViewer</class>
   <extends>QWidget</extends>
   <header>mpuviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="volumemakerui.qrc"/>
 </resources>
 <connections/>
</ui>
